src/rgw/rgw_civetweb.cc

```diff
@@ -101,7 +101,13 @@ int RGWCivetWeb::init_env(CephContext *cct)
     }
 
     const boost::string_ref name(header->name);
-    const auto& value = header->value;
+    string value = header->value;
```

Comment:
why copy to "value"?
can you modify in place (use: ``auto& value = header->value``)?